tests/test_pex_binary.py

```diff
@@ -167,3 +175,45 @@ def test_clp_prereleases_resolver():
     pex_builder = build_pex(reqs, options)
     assert pex_builder is not None
     assert len(pex_builder.info.distributions) == 3, 'Should have resolved deps'
+
+
+def test_build_pex():
+  with temporary_dir() as sandbox:
+    pex_path = os.path.join(sandbox, 'pex')
+    results = run_pex_command(['ansicolors==1.1.8', '--output-file', pex_path])
```

Comment:
Huh...all this time I thought `1.0.2` was the latest. Going to see if this newer version (still 2 years old) fixes the regex DeprecationWarning Pants CI is filled with.